scripts/utils/price-utils.js

```diff
@@ -139,9 +139,52 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     return true
   }
 
+  const checkNoProfitableOffer = async (order, exchange, currentMarketPrice) => {
+    const tokenInfo = fetchTokenInfoFromExchange(exchange, [order.buyToken, order.sellToken])
+
+    if (currentMarketPrice === undefined) {
+      currentMarketPrice = await getDexagPrice(
+        (await tokenInfo[order.buyToken]).symbol,
+        (await tokenInfo[order.sellToken]).symbol
+      )
+    }
+    // checks whether the order amount is negligible
+    if ((await valueInUSD(order, tokenInfo)) < 10) {
+      return true
+    }
+
+    // multiplicator is used to counter rounding errors
+    const multiplicator = 1000000000
+    const marketPrice = new BN(currentMarketPrice * multiplicator)
+    const orderPrice = order.priceNumerator
+      .mul(new BN(10).pow(new BN((await tokenInfo[order.sellToken]).decimals)))
+      .div(new BN(10).pow(new BN((await tokenInfo[order.buyToken]).decimals)))
+      .mul(new BN(multiplicator))
```

Comment:
This shouldn't work for USDC/DAI, since 10^9 * 10*6 / 10**18 < 1, and everything is rounded to zero.